src/hooks/useStakingValidator.ts

```diff
@@ -54,6 +55,13 @@ export const useStakingValidator = () => {
           .includes(validator.operatorAddress.toLowerCase())
       ) ?? [];
 
+    const unbondingValidators =
+      response?.validators.filter((validator) =>
+        unbondingDelegations
+          ?.map((d) => d.validator)
+          .includes(validator.operatorAddress.toLowerCase())
```

Comment:
[non-blocking] May be stylistic, but I find it easier to read if we convert the unbonding delegations into a set or map outside of this callback and call unbondingDelegstionsSet.has(...) or something like that.

Also technically more efficient but that doesn't really matter in this case

Reply:
nit, I wonder if it's worth pulling out a helper function of `includesValidator` somewhere since we call `.includes(validator.operatorAddress.tolowerCase()` 3 times in this file

Reply:
Updated to use `some` lemme know if that helps